GameLift/Realtime/app.js

```diff
@@ -41,155 +40,170 @@ function init(rtSession) {
+  // eslint-disable-next-line no-unused-vars
+  session.getPlayers().forEach((player, playerId) => {
+    session.sendReliableMessage(outMessage, player);
+  });
+  logger.info('Ending Game Session');
 }
 
 // Return true if the process is healthy
 function onHealthCheck() {
-    return true;
+  return true;
 }
 
 // On Player Connect is called when a player has passed initial validation
 // Return true if player should connect, false to reject
 function onPlayerConnect(connectMsg) {
-    // Perform any validation needed for connectMsg.payload, connectMsg.peerId
-	if (activePlayers >= 8)
-		return false;
-	
-	return true;
+  logger.info(`ConnectMSG Information: ${connectMsg}`);
+  // Perform any validation needed for connectMsg.payload, connectMsg.peerId
+  if (activePlayers >= 8) { return false; }
+
+  return true;
 }
 
 // Called when a Player is accepted into the game
 function onPlayerAccepted(player) {
-	logger.info("Player " + player.peerId + " accepted");
-    // This player was accepted -- let's send them a message
-    const msg = session.newTextGameMessage(OP_CODE_PLAYER_ACCEPTED, player.peerId,
-                                             "Player " + player.peerId + " accepted");
-    
-	session.sendReliableMessage(msg, player.peerId);
-    activePlayers++;
+  logger.info(`Player ${player.peerId} accepted`);
+  // This player was accepted -- let's send them a message
+  const msg = session.newTextGameMessage(OP_CODE_PLAYER_ACCEPTED, player.peerId,
+    `Player ${player.peerId} accepted`);
+
+  session.sendReliableMessage(msg, player.peerId);
+  activePlayers += 1;
 }
 
 // On Player Disconnect is called when a player has left or been forcibly terminated
-// Is only called for players that actually connected to the server and not those rejected by validation
-// This is called before the player is removed from the player list
+// Is only called for players that actually connected to the server and not those rejected by
+// validation. This is called before the player is removed from the player list
 function onPlayerDisconnect(peerId) {
-	logger.info("Player " + peerId + " disconnected");
-
-    // send a message to each remaining player letting them know about the disconnect
-    const outMessage = session.newTextGameMessage(OP_CODE_PLAYER_DISCONNECTED,
-                                                session.getServerId(),
-                                                "Player " + peerId + " disconnected");
-    session.getPlayers().forEach((player, playerId) => {
-        if (playerId != peerId) {
-            session.sendReliableMessage(outMessage, peerId);
-        }
-    });
-    activePlayers--;
+  logger.info(`Player ${peerId} disconnected`);
+
+  // send a message to each remaining player letting them know about the disconnect
+  const outMessage = session.newTextGameMessage(OP_CODE_PLAYER_DISCONNECTED,
+    session.getServerId(),
```

Comment:
Perhaps in the future we should change this so that the peerId is the sender? That way we don't have to parse out which peer is the one that disconnected from the message itself.

Reply:
Yup, I'll change that.